File: tests/fixtures/openai-realtime/openai-realtime-verify-deprecated-session-fields-fixed/01-no-temperature.ts
const callerConfigMsg = {
  type: 'session.update',
  session: {
    modalities: ['text', 'audio'],
    instructions: 'translate from Spanish to English',
    input_audio_format: 'g711_ulaw',
    output_audio_format: 'g711_ulaw',
    turn_detection: { type: 'server_vad' },
  },
};

export default callerConfigMsg;

File: tests/fixtures/openai-realtime/openai-realtime-verify-deprecated-session-fields-fixed/01_no_temperature.py
import json


async def configure_caller_session(ws):
    await ws.send(
        json.dumps(
            {
                "type": "session.update",
                "session": {
                    "instructions": "Translate caller audio to English.",
                    "turn_detection": {"type": "server_vad"},
                },
            }
        )
    )
